src/utils/todo/task-filing/content-loader.ts

```diff
@@ -105,17 +105,17 @@ export const getDraftTitle = (filing: TaskToDoI): string => {
     case FilingTypes.CONTINUATION_OUT:
       return FilingNames.CONTINUATION_OUT
     case FilingTypes.CONTINUATION_IN:
-      return FilingNames.CONTINUATION_IN_APPLICATION
+      return FilingNames.CONTINUATION_AUTHORIZATION
```

Comment:
There is both a continuation in Authorization and a continuation in Application (same filing type with different statuses). I think this needs to handle that.

Reply:
I stand corrected. This does work correctly:

![image](https://github.com/user-attachments/assets/909dc33a-7c3b-4968-a5dd-3057577c2e9b)
https://business-dashboard-dev--pr-97-5obencu4.web.app/TTk5dZdqlE

Reply:
Can you point me to the code that displays "continuation application", please?

Reply:
The `getTitle` function in `utils/todo/task-filing/content-loader.ts` is loading the title for todo items.  It calls the ` filingTypeToName`.

Reply:
Oh, I see. The code above is `getDraftTitle`, while the other code is `getTitle`.

So there are 2 separate places that the title is computed? Here and in https://github.com/bcgov/business-dashboard-ui/blob/main/src/utils/todo/task-filing/helper.ts?

DRY if possible, please!

Reply:
These three functions look similar but they return different values. 

`getTitle` is loading the 'title' field of a todo item (e.g. "BC Limited Company Continuation Application" in the above screenshot).

`getDraftTitle` is loading the `draftTitle` field of a todo item. This value is used in multiple confirmation dialogs, as well as the expansion content for pending online banking (`PaymentPendingOnlineBanking.vue`). 

The one (`filingTypeToName`) in https://github.com/bcgov/business-dashboard-ui/blob/main/src/utils/todo/task-filing/helper.ts  is a helper function that is called inside the above two functions. It is also called in other places.

Reply:
OK, fine for now.